src/Tests/With_UTF8_BOM.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.IO;
+using System.Text;
+using NServiceBus.MessageInterfaces.MessageMapper.Reflection;
+using NServiceBus.Newtonsoft.Json;
+using NUnit.Framework;
+
+[TestFixture]
+public class With_UTF8_BOM
+{
+
+    [Test]
+    public void Run()
+    {
+        var messageMapper = new MessageMapper();
+        var serializer = new JsonMessageSerializer(messageMapper, null, null, null, null);
+
+        var serialized = new UTF8Encoding(true).GetBytes($"{{\"{nameof(SimpleMessage.SomeProperty)}\":\"John\"}}");
```

Comment:
When I debug the test and look at the contents of this array, it doesn't actually have a BOM, so this test currently isn't testing what it's saying it does.